packages/gatsby-plugin-firebase/README.md

```diff
@@ -9,9 +9,8 @@ Provides drop-in support for Firebase
 
 - [Installation](#installation)
 - [Usage](#usage)
-  - [1. Set up environment variables](#1-set-up-environment-variables)
-  - [2. Register Gatsby plugin](#2-register-gatsby-plugin)
-  - [3. Use Firebase](#3-use-firebase)
+  - [1. Register Gatsby plugin](#2-register-gatsby-plugin)
```

Comment:
Can you revert back to the original README? You can add a section/note saying that if someone doesn't want to use environment variables, they can pass the keys via plugin option. However, the env variables would still be the recommended way.